newdocs/source/contributors.rst

```diff
@@ -0,0 +1,16 @@
+Contributors
+=================================
+
+* `Haris Shuaib <https://github.com/hshuaib90>`_
+* `Neil Heraghty <mailto:neil.heraghty@nhs.net>`_
+* `Laurence Jackson <https://github.com/laurencejackson>`_
+* `Tom Roberts <https://github.com/tomaroberts>`_
+* `Lucrezia Cester <https://github.com/Lucrezia-Cester>`_
+* `Paul Wilson <https://github.com/pcw24601>`_
+* `Yassine Azma <https://github.com/YassineRMH>`_
+* `Haleema Al Jazzaf <https://github.com/heyhaleema>`_
+* `Sophie Ratkai <https://github.com/sophie22>`_
+* `Ryan Satnarine <https://github.com/rs-sprout98>`_
+* `Jane Ansell <https://github.com/ansellj>`_
+* `Jamie Small <https://github.com/JamieSmall>`_
+* `Elizabeth Stamou <https://github.com/elizaGSTT>`_
```

Comment:
Can you also add some extra contributors:

Francesco Padormo <https://github.com/francescopadormo>
Sian Culley <https://github.com/superresolusian>